skxray/roi.py

```diff
@@ -229,3 +231,87 @@ def ring_edges(inner_radius, width, spacing=0, num_rings=None):
     edges = np.cumsum(steps).reshape(-1, 2)
 
     return edges
+
+
+def divide_pies(image_shape, radius, calibrated_center,
```

Comment:
`pie_slices(edges, slicing, center, shape, theta0=0)` where `slicing` can be an integer (`slicing=8` gives 8 equal-sized slices, like a pizza) or a list of angles in degrees like `[0, 70, 190]`. In either case `theta0` should be added to the resultant angles to provide an easy way to adjust the alignment.